src/library.js

```diff
@@ -2700,6 +2713,14 @@ LibraryManager.library = {
   emscripten_longjmp: function(env, value) {
     _longjmp(env, value);
   },
+#if WASM_BACKEND == 1
+  // We need this wrapper from llvm wasm backend, but this is generated only
+  // in asm.js compilation, so.
```

Comment:
In asm.js, emscripten_longjmp__wrapper is generated here:
https://github.com/kripken/emscripten/blob/incoming/emscripten.py#L540
But actually I don't understand the purpose of this make_table and fix_item functions, and why asm.js generates this wrapper in the first place.

_longjmp exists in JS, so it does not call back into asm/wasm.
https://github.com/kripken/emscripten/blob/incoming/src/library.js#L2695

Yes, I don't know how much details I should add. I can add all the details I described here
https://github.com/kripken/emscripten/pull/4524#discussion_r76551772
as a comment, if you and Alon think it is better.